Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3292,6 +3292,8 @@ void LocalFrameView::unobscuredContentSizeChanged()
 {
 #if PLATFORM(IOS_FAMILY)
     updateTiledBackingAdaptiveSizing();
+    if (auto* document = m_frame->document())
+        document->scheduleFullStyleRebuild();
```

Comment:
This might not be sufficient. Layout is generally but not always occurring after a full style rebuild.

Maybe I want `layoutContext().layout();` instead?